package-lock.json

```diff
@@ -0,0 +1,388 @@
+{
```

Comment:
You have both a package-lock.json and a yarn.lock file. That means you ran both npm and yarn. If you do that by accident, make sure you delete either the package-lock.json to get rid of npm or the yarn.lock file to get rid of yarn.